scripts/src/tests/services/orders.spec.ts

```diff
@@ -653,7 +653,7 @@ describe("test v2 orders", async () => {
 		const orders2 = await createOrdersForUser(user, deviceId2, app, { divideBy: 2 });
 		expect(orders2.length).toBeGreaterThan(0);
 
-		expect((await user.getWallets()).count).toBe(2);
+		expect((await user.getWallets()).count).toBe(1);
```

Comment:
@nitzantomer FYI had to change your test

Reply:
makes sense!
good.